.github/workflows/pylint_check.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+
+TOXINI_FILE = Path("tox.ini")
+PYPROJECTTOML_FILE = Path("pyproject.toml")
+GLOBAL_DISABLES = {
+    "invalid-name",
+    "no-else-raise",
+    "no-else-return",
+}
+
+
+# Parse ./tox.ini
+tox_lines = TOXINI_FILE.read_text().splitlines()
+tox_pylint_lines = [l for l in tox_lines if l.strip().startswith("pylint")]
+tox_disables = {i for l in tox_pylint_lines for i in l.split('"')[1].split(",")}
+
+
+# Parse ./pyproject.toml
+pyproject_lines = PYPROJECTTOML_FILE.read_text().splitlines()
+is_in_disable_string = False
+pyproject_disables = set()
+for line in pyproject_lines:
+    if is_in_disable_string and line.startswith('"""'):
+        is_in_disable_string = False
+
+    if is_in_disable_string and line:
+        pyproject_disables.add(line.strip().strip(","))
+
+    if line.startswith("disable"):
+        is_in_disable_string = True
+
+
+# Check correctness and raise errors
+try:
+    in_pyproject_butnot_tox = pyproject_disables.difference(tox_disables).difference(
+        GLOBAL_DISABLES
+    )
+    assert not in_pyproject_butnot_tox
+except AssertionError:
+    raise Exception(
+        f"""
```

Comment:
I think the most important point for the review could be how informative these error messages are. Please make sure that you understand them and that you have an idea of what you would do if you ever get this error.